railties/test/application/middleware/sendfile_test.rb

```diff
@@ -23,49 +23,17 @@ def index
       end
     end
 
-    # x_sendfile_header middleware
-    test "config.action_dispatch.x_sendfile_header defaults to nil" do
-      make_basic_app
-      simple_controller
-
-      get "/"
-      assert_not last_response.headers["X-Sendfile"]
-      assert_equal File.read(__FILE__), last_response.body
-    end
-
-    test "config.action_dispatch.x_sendfile_header can be set" do
+    # x_sendfile_header middleware deprecated
+    test "config.action_dispatch.x_sendfile_header cannot be set" do
       make_basic_app do |app|
         app.config.action_dispatch.x_sendfile_header = "X-Sendfile"
```

Comment:
We need to deprecate this option now that it has no behavior. can you do that?

Reply:
@rafaelfranca,  Thank you for the reply

Do you mean something like this https://github.com/rails/rails/pull/44556/commits/a304235ff835d9d0b1da3004fb7f6243ec0062be?